app/config.go

```diff
@@ -243,10 +243,10 @@ func (a *App) ensureInstallationDate() error {
 		return nil
 	}
 
-	result := <-a.Srv.Store.User().InferSystemInstallDate()
+	createdAt, err := a.Srv.Store.User().InferSystemInstallDate()
```

Comment:
Probably here makes sense to name the variable something like `installDate`

Reply:
ok @jespino I renamed the var name ;)